build.sbt

```diff
@@ -284,7 +284,7 @@ ThisBuild / githubWorkflowEnv ++= Map(
   "ALPAKKA_S3_AWS_CREDENTIALS_SECRET_ACCESS_KEY" -> "${{ secrets.AWS_SECRET_KEY }}"
 )
 
-ThisBuild / githubWorkflowJavaVersions := List("openjdk@1.11")
+ThisBuild / githubWorkflowJavaVersions := List(JavaSpec.temurin("11"))
```

Comment:
The latest update to sbt-github-actions has changed how you specify Java versions. It is now properly typed (as you can tell from the `JavaSpec.temurin` and rather using OpenJDK it uses temurin instead.

According to http://whichjdk.com/ its a high quality JDK release

> Recommendation: Adoptium Eclipse Temurin OpenJDK builds are highly recommended.